src/examples/recognised-overseas-pension-schemes-notification-list/example/index.njk

```diff
@@ -0,0 +1,87 @@
+---
+  layout: layout-example.njk
+---
+
+<h3 class="govuk-heading-l">Austria</h3>
+
+<div class="call-to-action">
+<p class="govuk-body">The requirements to be a ROPS changed from 6 April 2017. You’ll need to check that the scheme you’re transferring to on or after that date meets the new requirements.<br><br></p>
```

Comment:
For an example we shouldn't be encouraging `<br>` elements to add whitespace.

Reply:
Yes, you're right. They seem to have been added intentionally to the current live page. I've now removed and they're not really necessary.